src/protocol/webdriver-bidi.ts

```diff
@@ -1307,7 +1307,10 @@ export namespace Script {
     arguments?: [...Script.LocalValue[]];
     resultOwnership?: Script.ResultOwnership;
     serializationOptions?: Script.SerializationOptions;
-    this?: Script.LocalValue;
+    this?: Script.LocalValue /**
```

Comment:
@jrandolf Same issue as the other comments Prettier 3 mangle this

Reply:
File bug in CDDL repo?